package/skelcd-control-MicroOS.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Jan 25 09:39:34 UTC 2022 - Alberto Planas Dominguez <aplanas@suse.com>
+
+- Remove ALPHA tag for the Keylime services
+- Reference the new place for the configuration file
```

Comment:
A reference is needed to please our internal Jenkins.

Use ghh#yast/skelcd-control-MicroOS#49

Reply:
also version bump is needed, otherwise it won't be submited

Reply:
IIRC the accepted format is `gh#skelcd-control-MicroOS#49` ...

Reply:
Just for the record, the long format is valid too. See https://github.com/openSUSE/packaging_rake_tasks/pull/54/files#diff-8045d97188605edfd44f8902c7fc3d70ad506f0cc0dc38f2b982f9dccf08e091R50


In fact, I used it yesterday at https://github.com/yast/skelcd-control-MicroOS/pull/48